test/scripts/filters/backtick_code_block.js

```diff
@@ -195,6 +195,27 @@ describe('Backtick code block', () => {
     data.content.should.eql('<escape>' + expected + '</escape>');
   });
 
+  it('only wrap with pre and code', () => {
+    hexo.config.highlight.exclude_languages = ['js'];
+    const data = {
+      content: [
+        '``` js',
+        code,
+        '```'
+      ].join('\n')
+    };
+
+    const expected = highlight(code, {
+      lang: 'js',
+      gutter: false,
+      hljs: true,
+      wrap: false
+    });
+
+    codeBlock(data);
+    data.content.should.eql('<escape>' + expected + '</escape>');
+  });
```

Comment:
need to reset `hexo.config.highlight.exclude_languages` at the end of this test.

Reply:
Well, `beforeEach()` has made a deep clone from the default config, I don't think it is necessary to reset that value.